script/testing/reporting/parsers/parse_data.py

```diff
@@ -49,44 +48,52 @@ def parse_microbenchmark_data(artifact_processor_comparison):
         test_name (str): The name of the specific benchmark test
         metrics (dict): The metrics gathered from the result of the test
     """
-    env_metadata = parse_jenkins_env_vars()
     metadata = parse_standard_metadata()
     test_suite, test_name, metrics = parse_microbenchmark_comparison(
         artifact_processor_comparison)
     return metadata, test_suite, test_name, metrics
 
 
 def parse_standard_metadata():
-    """ Gather the standard metadata infromation from Jenkins and the DBMS.
-    This will get the DB version from a file. If there is a better way to
-    get that information from the test then that is advisible 
-    (like OLTPBench)"""
-    env_metadata = parse_jenkins_env_vars()
-    metadata = {**env_metadata, **parse_db_metadata()}
-    return metadata
+    """
+    Gather the standard metadata information from Jenkins and the DBMS.
+
+    Returns
+    -------
+    The metadata obtained from Jenkins and the DBMS.
 
+    Warnings
+    --------
+    Underlying implementation is hacky right now.
+    """
+    return {**_parse_jenkins_env_vars(), **_parse_db_metadata()}
 
-def parse_jenkins_env_vars():
-    """ get some metadata from the environment values that Jenkins has 
-    populated and from the operating system"""
-    jenkins_job_id = os.environ['BUILD_ID']
-    git_branch = os.environ['GIT_BRANCH']
-    commit_id = os.environ['GIT_COMMIT']
-    os_version = ' '.join(distro.linux_distribution())
-    os_cpu_number = os.cpu_count()
+
+def _parse_jenkins_env_vars():
+    """
+    Parse environment variables from Jenkins and the OS.
+
+    Returns
+    -------
+    metadata : dict
+        Metadata about the Jenkins environment.
+        WARNING: Note that cpu_socket is a completely garbage value.
+        TODO(WAN): I'd remove cpu_socket except I'm afraid of breakages.
```

Comment:
It won't break anything. We added that cause we hoped we could figure out how to get a meaningful value eventually.

Reply:
punting, leaving as comment